newIDE/app/src/ObjectsList/index.js

```diff
@@ -261,15 +261,15 @@ export default class ObjectsListContainer extends React.Component<
 
   _duplicateObject = (objectWithContext: ObjectWithContext) => {
     this._copyObject(objectWithContext);
-    this._editName(this._paste(objectWithContext));
+    this._pasteAndRename(objectWithContext);
   };
 
-  _pasteObject = (objectWithContext: ObjectWithContext) => {
+  _pasteAndRename = (objectWithContext: ObjectWithContext) => {
```

Comment:
I've used an explicit name "_pasteAndRename" because otherwise the difference between paste and "pasteObject" is not clear